templates/repo/diff/new_review.tmpl

```diff
@@ -22,9 +22,21 @@
 					</div>
 				{{end}}
 				<div class="ui divider"></div>
-				<button type="submit" name="type" value="approve" {{if and $.IsSigned ($.Issue.IsPoster $.SignedUser.ID)}} disabled {{end}} class="ui submit green tiny button btn-submit">{{$.locale.Tr "repo.diff.review.approve"}}</button>
+				{{if and $.IsSigned ($.Issue.IsPoster $.SignedUser.ID)}}
+					<span class="ui tooltip" data-content="{{$.locale.Tr "repo.diff.review.self_approve"}}">
+						<button type="submit" name="type" value="approve" disabled class="ui submit green tiny button btn-submit">{{$.locale.Tr "repo.diff.review.approve"}}</button>
+					</span>
```

Comment:
These `span` wrappings can be removed, just put the `tooltip` class and the `data-content` on the button itself.

Reply:
As it is a issue https://github.com/atomiks/tippyjs/issues/286 and fixed in later release.


I tried to do so  but we have to programatically handle the  in tippyjs
something like this `instance.enable();`

https://atomiks.github.io/tippyjs/v6/methods/#enable